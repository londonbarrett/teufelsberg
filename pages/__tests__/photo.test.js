/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import Photo from '../photo.js';

describe('Photo Page Snapshot Testing', () => {
  it('Renders Photo Page correctly', () => {
    const component = renderer.create(<Photo router={{ query: { id: 'image_01' } }} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
