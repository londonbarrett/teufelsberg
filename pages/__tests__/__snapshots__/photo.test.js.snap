// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo Page Snapshot Testing Renders Photo Page correctly 1`] = `
<div>
  <a
    href="/"
    onClick={[Function]}
  >
    X
  </a>
  <img
    alt="Mobile.de"
    src="https://image_01_27.jpg"
  />
</div>
`;
